src/Shared/HttpExtensions.cs

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+using Microsoft.AspNetCore.Http;
+
+internal static class HttpExtensions
+{
+    internal static string UrlEncodedFormContentType = "application/x-www-form-urlencoded";
+    internal static string MultipartFormContentType = "multipart/form-data";
```

Comment:
Could/should these be constants? Then you could test against the length of the shortest one lower down and avoid the 19 magic number?